app/views/groups/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<nav class = "nav-top-two">
+    <a href="/"><i class="fas fa-home" aria-hidden="true"></i></a>
+    <p id= "no-padding">CATEGORIES</p>
+    <%= link_to destroy_user_session_path, method: :delete, class: "links" do %>
+        <i class="fas fa-sign-out-alt"></i>
+    <% end %>
+</nav>
+
+<div class="extra">
+    <% if @groups.exists? %>
+    <ul class = "group-ul">
+        <% @groups.each do |group| %>
+          <li class = "group-list">
+            <div class="list-items">
+                <div class = "group-holder">
+                    <img class="w-25" src="<%= group.icon%>" alt="No IMG">
+                   <p class = "title"> <%= link_to group.name, group_path(group) %> </p>
+                </div>
+                <p class = "currency title"><%= number_to_currency(group.money_tracks.sum(&:amount))%></p>
+            </div>
+          </li>
```

Comment:
- [ ] For the item cards, please let us stick to the layout, font-weight and color of the Behance design. Please see attached image below and try to follow the design as closely as possible 👇 
### Design Guideline
![image](https://github.com/Marcraphael12/rails-budget-app/assets/46839548/3fd0afc8-cc60-4390-a654-78ac0270910a)
### Your Design
![image](https://github.com/Marcraphael12/rails-budget-app/assets/46839548/d5ca28fb-e00b-468d-a52b-f2f102a89362)

Reply:
Thanks for noticing??? it's alll fixed